client/resource_group/controller/limiter.go

```diff
@@ -296,18 +298,22 @@ func (lim *Limiter) Reconfigure(now time.Time,
 ) {
 	lim.mu.Lock()
 	defer lim.mu.Unlock()
-	log.Debug("[resource group controller] before reconfigure", zap.Float64("NewTokens", lim.tokens), zap.Float64("NewRate", float64(lim.limit)), zap.Float64("NotifyThreshold", args.NotifyThreshold))
-	now, _, tokens := lim.advance(now)
-	lim.last = now
-	lim.tokens = tokens + args.NewTokens
+	log.Debug("[resource group controller] before reconfigure", zap.Float64("NewTokens", lim.tokens), zap.Float64("NewRate", float64(lim.limit)), zap.Float64("NotifyThreshold", args.NotifyThreshold), zap.Int64("burst", lim.burst))
+	if args.NewBurst < 0 {
+		lim.tokens = args.NewTokens
```

Comment:
The annotation for the limiter field `now` is `last is the last time the token field of the limiter was updated`
So for this change do we need to update comment or  Will this cause some unexpect behavior?

Reply:
I changed it and will update `last` here as well